NanamiL3.java

```diff
@@ -0,0 +1,122 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+// import packages to read user input from console
+
+public class NanamiL3 {
```

Comment:
Do consider proper naming instead of NanamiL3 or Nanami_L0, L3 or L1 can be in git tag/commits instead.